src/main/java/uk/gov/hmcts/reform/jobscheduler/contollers/JobsController.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.jobscheduler.contollers;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.jobscheduler.model.Job;
+import uk.gov.hmcts.reform.jobscheduler.services.JobsService;
+
+import java.net.URI;
+
+import static org.springframework.http.ResponseEntity.created;
+import static org.springframework.web.servlet.support.ServletUriComponentsBuilder.fromCurrentRequest;
+
+@RestController
+@RequestMapping(path = "/jobs")
+public class JobsController {
```

Comment:
How about `JobService`?

Reply:
I like plural, the 'resource' that we access is jobs imho.
Personal preference I guess